docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md

```diff
@@ -0,0 +1,199 @@
+
+![](imgs/2_mambaforge_install_3.png)
+
+Install Mambaforge into the default location:
+
+![](imgs/2_mambaforge_install_4.png)
+ 
+In the next step we recommend to additionally tick `Add Mambaforge to my Path`. If you don’t add it to the [Path](https://janelbrandon.medium.com/understanding-the-path-variable-6eae0936e976), Conda and Mamba would not work from any Terminal Window. Click `Install`
+
+![](imgs/2_mambaforge_install_5.png)
+
+Click `Next` at the next window and you arrive here. Click `Finish` to exit the setup:
+
+![](imgs/2_mambaforge_install_7.png)
+
+Great! You are ready to start coding! 👍 
+To see how to use Mambaforge, jump [here](#using-mambaforge)
+
+### Installation on Mac OS
+
+First, you open your Terminal. You can find it using the Spotlight Search and typing _Terminal_ like this:
+
+![](imgs/2_mac_install_1.png)
+
+The opened Terminal should look like this: 
+
+![](imgs/2_mac_install_2.png)
+
+Then type these two lines to start the installation
+
+```json
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
+```
+
+```json
+bash Mambaforge-$(uname)-$(uname -m).sh
+```
+
+You  just follow the instructions, press `Enter` and type _yes_ like here:
+
+![](imgs/2_mac_install_3.png)
+
+When you see this:
+
+![](imgs/2_mac_install_4.png)
+
+you are finished! Close and reopen the Terminal now. Happy coding! 👍
+
+## Using Mambaforge 
+
+Now we use Mambaforge by opening the command line. If you are not familiar with the command line yet, you can check out Roberts tutorial [here](https://www.youtube.com/watch?v=MOEPe9TGBK0&t=1146s). 
+
+To open the Command Prompt in Windows, press the `Windows button`, type _cmd_ and press `Enter`. The Mac OS users should already know how to open the Terminal ;-)
+
+When you open the command line, it should look like this: 
+
+![](imgs/3_envs_1.png)
+
+You are now located in the base environment. It is the default environment and includes beside a Python installation the core Conda libraries and dependencies.
+
+> **_Strong recommendation:_** Never install any packages into the base environment! 
+
+The reason for this is that incompatibilities between packages can occur. Robert demonstrated this [here](https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/). If this happens in any environment apart from the base environment it is no problem. You can delete the environment, recreate it and start again. If this happens in the base environment, you need to delete and reinstall Mambaforge. 
+
+## Creating a new environment 
+You can create a new environment typing the following command into the Command Prompt: 
+
+```json
+mamba create -n my_first_env devbio-napari python=3.9 -c conda-forge 
```

Comment:
Hm...I think the command to create new environments should just be that:  A command to create a new (blank) environment. This one already installs quite some stuff, which is absolutely legit in our usecase. In some cases, I may need things in my to-be-created environment, which may not necessarily be compatible with devbio-napari. In that case I think it would be nice to tell a collaborator: "Please follow this tutotial up until there" instead of "Please follow this tutorial up until X and change the command in line Y to Z"

Reply:
This blog post is about the use case that people want to collaborate with us. Thus, they should install devbio-napari. Otherwise we will have to answer install-package-X questions until the end of days. I think the text is ok as it is. Don't make it more complicated

Reply:
I'm not saying to NOT install devbio-napari. I'm just suggesting to put this into two separate commands.